### ISSUE TITLE
Export to paste buffer

### ISSUE THREAD

**Issue by philrz** [brimdata/zui#2982]
## tl;dr

To make it even easier to share data from Zui with other tools, we could add a feature to allow Export directly to the paste buffer.

## Details

At the time this issue is being filed, Zui is at commit e275ddd.

In a recent [community Slack message](https://brimdata.slack.com/archives/C010DR0HHMF/p1705703228047349?thread_ts=1704480595.539399&cid=C010DR0HHMF), a user said the following:

>i have been running zui events and taking the export csv for now to do the comparisons with filters at the spreadsheet level. eventually i will get smart enough to use the zed language more effectively

While we definitely want users to learn the Zed language and be as productive as possible within the app itself, there will inevitably be times it makes sense for users to take the data from Zui into other tools, so we'd like to make it as easy as possible for them to do that. Thinking specifically about spreadsheets, when we recently added TSV import/export options, spreadsheet tools were a big motivator since our community users helped make us aware how cut & paste in/out of those tools is TSV-based.

Right now a user can use the existing Export feature to put TSV into a file, then import that file into the spreadsheet tool or cut & paste its contents once the file is created. However, seeing the user's message made me realize we could save them even more clicks if Zui's Export feature had a way to send data directly into the paste buffer.

Just to brainstorm ideas, one approach might be to add an additional button to the **Export Results** pop-up, e.g., change the current **Export** button to **Export to File** and add a new **Export to Paste Buffer** button.

![image](https://github.com/brimdata/zui/assets/5934157/c2324f60-cecf-4dbf-8a9b-a3ba4ba12df7)

As @jameskerr is currently putting in a shortcut for opening the **Export Results** pop-up (#2979), another fancy variation comes to mind: Perhaps the app could cache the last selected format from the **Export Results** pop-up, which would then allow us to offer a shortcut that would bring the current query results into the paste buffer in that last-selected format without even having to pause and click at the **Export Results** pop-up.

**Comment by philrz** [comment]
Verified in Zui commit 1365bbd.

As shown in the attached video, clicking **Export** now offers a **Copy to Clipboard** button that's an alternative to the export-to-file we've always had and the new export-to-pool option that was also added. What's shown here is export in TSV format for immediate paste into a spreadsheet tool such as the community user originally requested. There's also a keyboard shortcut for bringing up **Export**, a keyboard shortcut for selecting the **Copy to Clipboard** option, and **Esc** can always be hit to dismiss the **Export** modal, so if users are doing frequent exports to the paste buffer this also offers them a keyboard-centric approach.

https://github.com/brimdata/zui/assets/5934157/1db169ef-f54f-42ff-b40b-ef11ddab2ee9

Thanks @jameskerr!
